src/components/molecules/DropdownField/DropdownField.tsx

```diff
@@ -15,23 +15,33 @@ export interface IDropdownFieldProps extends IField<HTMLSelectElement> {
   name: string;
 }
 
+export interface ILabelProps {
+  withHelpText?: boolean;
```

Comment:
Might be good to assign a default value to `withHelpText` so its always a boolean and not a boolean or undefined

Reply:
Hmm I don't see the benefit of it vs the current implementation 🤔

Reply:
So you narrow down the type. It strictly becomes a boolean.